app/src/pages/common/settingsPage/notificationsTab/notificationsTab.jsx

```diff
@@ -18,33 +18,59 @@ import React, { Component, Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import classNames from 'classnames/bind';
-import { injectIntl, defineMessages } from 'react-intl';
+import track from 'react-tracking';
+import { injectIntl } from 'react-intl';
+import { formatAttribute } from 'common/utils/attributeUtils';
 import { canUpdateSettings } from 'common/utils/permissions';
+import PlusIcon from 'common/img/plus-button-inline.svg';
+import { SETTINGS_PAGE_EVENTS } from 'components/main/analytics/events';
+import { GhostButton } from 'components/buttons/ghostButton';
 import {
   updateProjectNotificationsConfigAction,
   projectNotificationsCasesSelector,
   projectNotificationsEnabledSelector,
 } from 'controllers/project';
 import { isEmailIntegrationAvailableSelector } from 'controllers/plugins';
+import { showModalAction } from 'controllers/modal';
 import { activeProjectRoleSelector, userAccountRoleSelector } from 'controllers/user';
-import { NotificationsEnableForm } from './notificationsEnableForm';
-import { NotificationCaseList } from './notificationCaseList';
 import { NoCasesBlock } from './../noCasesBlock';
-import { AddNewCaseButton } from './addNewCaseButton';
 import styles from './notificationsTab.scss';
+import { ListHeader } from '../notificationsPatternAnalysisTabs/listHeader';
```

Comment:
If these are common components, it's better to move them to more common place. Maybe `pages/common/settingsPage/notificationList`